src/components/VisuItem/VisuItem.tsx

```diff
@@ -16,17 +16,23 @@ export const VisuItem = ({
 
   return (
     <ItemWrapper>
-      <Input
-        type="radio"
-        id={id}
-        name={name}
-        checked={checked}
-        value={id}
-        disabled={isLoading}
-        onChange={() => onChange(id)}
-        onClick={() => onClick(id, false)}
-      />
-      <Label htmlFor={id}>{name}</Label>
+      {isLoading ? (
+        <LoadingIcon id="isLoading" />
```

Comment:
```suggestion
        <LoadingIcon id="loading" size={18} />
```

To show the icon we want we need to access the `sprite.svg` file and check the ids inside this file. The sprite.svg file contains all the svg icons we use and they are marked with an "id" property. Because of that we need to change this "_isLoading_" to _"loading"_, which is an id that exists on the sprite file.

That's why we won't need all the CSS from the Icon component that was on the styles file.